Vizzuality/airqast
# Demos: open dropdowns toward center of the screen

This PR makes sure that when the user opens the dropdowns of the demos, the window doesn't scroll away from the demos.

## Testing instructions

1. Go to http://localhost:3000/products/emissions/demo
2. Open the year and month dropdowns

The dropdowns should open towards above and not below. You can test the other dropdowns: when comparing and the ones of the other demo too.

## Pivotal
[Task](https://www.pivotaltracker.com/story/show/160473631)
